### ISSUE TITLE
{{else}} block for empty results

### ISSUE THREAD

**Issue by esbanarango** [cibernox/ember-power-select#1370]
We're upgrading from `3.0.6` to `4.0.2`. 
We have this searchable power-select component, and we were using the {{else}} block for displaying custom html code when no results. After upgrading that block isn't showing anymore.
```hbs
{{#power-select
  searchEnabled=true
  search=(queue
            (action (mut searchName))
            (action (perform searchEmployeesFromLocationTask))
          )
  placeholder=" Type to search employees at other locations"
  labelPath="name"
  selected=changeset.name
  disabled=disabled
  onChange=(action updateEmployee) 
  as |employee|
}}
  {{employee.name}}  
{{else}}
  CUSTOM HTML CODE
{{/power-select}}
```
Apparently that was removed [here](https://github.com/cibernox/ember-power-select/blob/cd84c245cf1b550fca6f0b4590649487d6a0b6a0/tests/dummy/app/templates/snippets/the-list-6.hbs). What's the suggested approach now? Or how could we get this {{else}} block back?

Thank you.

**Comment by ybaldus** [comment]
At the moment the argument "noMatchesMessage" allows you to place simple text in there. If you need html code, you could write a custom modifier (https://github.com/ember-modifier/ember-modifier). Unfortunately a modifier can only be placed on the power select if it's rendered in place (it passes attributes to a basic dropdown which will only apply them if rendered in place). A solution to this would be wrapping the power select inside a div and placing the modifier on the div.

**Comment by cibernox** [comment]
Yeah, we had to give up that feature when updating to glimmer components. You still can do it by overriding the `optionsComponent`, but it's very inconvenient. I hope that named blocks will allow us to reintroduce this feature.
